packages/core/src/internal/utils/dom.ts

```diff
@@ -102,3 +102,13 @@ export function listenForAttributeChange(
 export function isVisible(element: HTMLElement) {
   return element?.offsetHeight > 0 && element?.hasAttribute('hidden') === false;
 }
+
+export function spanWrapper(nodeList: NodeListOf<ChildNode>): void {
+  nodeList.forEach(node => {
+    if (node.parentElement && node.textContent !== null && node.nodeType === 3) {
+      const spanWrapper = document.createElement('span');
+      spanWrapper.innerText = node.textContent;
+      node.parentElement.replaceChild(spanWrapper, node);
+    }
```

Comment:
This is close but should capture the node as a reference instead of replacing it that way framework bindings can continue updating the original node.

```typescript
    if (node.parentElement && node.textContent !== null && node.nodeType === 3) {
      const spanWrapper = document.createElement('span');
      const parent = node.parentElement;
      spanWrapper.appendChild(node);
      parent.appendChild(spanWrapper);
    }
```

https://stackblitz.com/edit/angular-ivy-hm3g1q?file=src%2Fapp%2Fapp.component.ts